ecl/hql/hqlgram2.cpp

```diff
@@ -12242,7 +12242,7 @@ extern HQL_API void parseModule(IHqlScope *scope, IFileContents * contents, HqlL
         E->Release();
     }
     if (ctx.hasCacheLocation())
-        moduleCtx.createCache(nullptr, false);
+        moduleCtx.createCache(nullptr, nullptr, false);
```

Comment:
why not passing contents?

Reply:
@ghalliday  I didn't pass contents as it isn't used if simplified expression is null.  I didn't want to mislead the reader in thinking it would be used or needed by createContents.  Should I still pass contents?

Reply:
safer to pass it - otherwise what happens if the function is changed to use it?